Source/Charts/Data/Implementations/Standard/BarChartData.swift

```diff
@@ -14,7 +14,7 @@ import CoreGraphics
 
 open class BarChartData: BarLineScatterCandleBubbleChartData
 {
-    public override init()
+    public required init()
```

Comment:
why many become required init() now?

Reply:
It's a requirement of `RangeReplaceableCollection`

Reply:
hmm.. My Dash API Doc does not show this to me

Reply:
From [Apple](https://developer.apple.com/documentation/swift/rangereplaceablecollection)

Reply:
![image](https://user-images.githubusercontent.com/4375169/34700674-34327846-f51f-11e7-8136-76ee7d83fb6b.png)
Found a Dash bug :)